components/email/src/Email.svelte

```diff
@@ -959,21 +1003,42 @@
                   </div>
                 {/if}
                 <div class="from-participants">
-                  {#if activeThread.messages.length >= 1 && activeThread.messages[activeThread.messages.length - 1].from.length}
-                    <span class="from-sub-section"
-                      >{activeThread.messages[activeThread.messages.length - 1]
-                        .from[0].name ||
-                        activeThread.messages[activeThread.messages.length - 1]
-                          .from[0].email}</span
-                    >
-                  {/if}
-                  {#if activeThread.messages.length > 1 && activeThread.messages[0].from.length && activeThread.messages[0].from[0].email !== activeThread.messages[activeThread.messages.length - 1].from[0].email}
-                    ,
-                    <span class="from-sub-section"
-                      >{activeThread.messages[0].from[0].name ||
-                        activeThread.messages[0].from[0].email}</span
-                    >
-                  {/if}
+                  <div class="participants-name">
+                    {#if activeThread.messages.length >= 1 && activeThread.messages[activeThread.messages.length - 1].from.length}
+                      <span class="from-sub-section"
+                        >{activeThread.messages[
+                          activeThread.messages.length - 1
+                        ].from[0].name ||
+                          activeThread.messages[
+                            activeThread.messages.length - 1
+                          ].from[0].email}</span
+                      >
+                    {/if}
+                    {#if activeThread.messages.length > 1 && activeThread.participants.length >= 2 && activeThread.messages[0].from.length && activeThread.participants[0].email !== activeThread.messages[activeThread.messages.length - 1].from[0].email}
+                      <span class="from-sub-section second"
+                        >, {activeThread.participants[0].name ||
+                          activeThread.participants[0].email}</span
+                      >
+                    {/if}
+                  </div>
+                  <div class="participants-count">
+                    <!-- If it is mobile, we only show 1 participant (latest from message), hence -1 -->
```

Comment:
nit - We could using magic numbers here by defining top-level constants for the number of mobile and desktop participants (this would also mean that we wouldn't need these comments here.

```
const MAX_DESKTOP_PARTICIPANTS = 1;
const MAX_MOBILE_PARTICIPANTS = 1;

...

{#if activeThread.participants.length >= MAX_MOBILE_PARTICIPANTS}
    <span class="show-on-mobile">
        &nbsp; &plus; {activeThread.participants.length - MAX_MOBILE_PARTICIPANTS}
    </span>
{/if}
```

Reply:
For sure!